lib/ts/BufferWriter.h

```diff
@@ -471,7 +491,7 @@ namespace bw_fmt
 
   /// This exists only to expand the index sequence into an array of formatters for the tuple type
   /// @a TUPLE.  Due to langauge limitations it cannot be done directly. The formatters can be
-  /// access via standard array access in constrast to templated tuple access. The actual array is
+  /// accessed via standard array access in constrast to templated tuple access. The actual array is
   /// static and therefore at run time the only operation is loading the address of the array.
```

Comment:
General observation:  comments are not using the full 132 character line width that is the ATS standard.

Reply:
The width for comments is still debated, I'm actually pushing the limits here as it is.

Reply:
The Double Secret LImits?

Reply:
Community standards.